src/scripts/scripts/zone/coilfang_resevoir/serpent_shrine/boss_lady_vashj.cpp

```diff
@@ -531,7 +531,7 @@ struct boss_lady_vashjAI : public ScriptedAI
                 if(Persuasion_Timer < diff)
                 {
                     MindcontrolEffect();
-                    Persuasion_Timer = 120000;
+                    Persuasion_Timer = urand(20000, 35000);
```

Comment:
I would check what the persuasion/mind control cooldown is before applying this. We may wish to avoid having 10 people mind controlled at once?